.pylintrc

```diff
@@ -237,7 +237,7 @@ indent-after-paren=4
 # String used as indentation unit: 4 spaces (no tab).
 indent-string='    '
 
-max-line-length=80    # Maximum number of characters on a single line.
+max-line-length=88    # Maximum number of characters on a single line.
```

Comment:
:thinking:

Reply:
Just sync with black defaults, but if we want to keep 80 I could also change the black value